contentgrid-spring-querydsl/src/main/java/com/contentgrid/spring/querydsl/mapping/UnsupportedCollectionFilterPredicateException.java

```diff
@@ -6,12 +6,10 @@
 
 @Getter
 public class UnsupportedCollectionFilterPredicateException extends UnsupportedOperationException{
-    private final QuerydslPredicateFactory<?, ?> predicateFactory;
```

Comment:
I'm not sure that this is unnecessary. It can be used to provide more details why this predicate does not work?

Reply:
Ok, this is not used - at this moment.

Then again, exceptions should be serializable. Would you prefer making this a transient field ?

Reply:
As discussed: we're removing the field.